tools/run_tests/run_tests.py

```diff
@@ -1254,9 +1262,12 @@ def isclose(a, b, rel_tol=1e-09, abs_tol=0.0):
 args = argp.parse_args()
 
 flaky_tests = set()
+test_times = {}
```

Comment:
if this is going to contain cpu info indexed by the test shortname, how about calling it `shortname_to_cpu'?

Reply:
Good advice, done.